lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p><code>age_range = [item["age_range"] for item in data]</code> už musí zvládnout :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
spíš pro dokreslení, znát to už musí, ale nemyslim, že je to nějak navíc

Reply:
<p>Když mně přijdou ty list comprehensions daleko víc pythonic než for cykly + append :)</p>